### ISSUE TITLE
Palettes

### ISSUE THREAD

**Issue by Paradox** [MiSTer-devel/Main_MiSTer#591]
Wondering if we can move Palettes out to their own directory, like was done for docs? 

/media/fat/Paletttes/(corename)

**Comment by birdybro** [comment]
Palettes are core dependent and are supposed to be automatically downloaded into that core's folder, e.g.:

```
.../games/NES/palette.zip
```

They are downloaded already as their own folder here --> https://github.com/MiSTer-devel/Distribution_MiSTer/tree/main/games/NES/Palettes

**Comment by Paradox** [comment]
I don't know why you feel the need to answer my questions like I'm an idiot. I know all of that, but, for one thing, whoever wrote the download script can't detect where the actual cores are, and keeps downloading them to /media/fat, even though I have zero of my roms there, and the palettes would never be used, if I didn't manually move them, myself, but, it redownloads them, as well as the drivers and vhds and everything, every damn time I update.

**Comment by thehughhefner** [comment]
> I don't know why you feel the need to answer my questions like I'm an idiot. I know all of that, but, for one thing, whoever wrote the download script can't detect where the actual rom directories are, and keeps downloading them to /media/fat, even though I have zero of my roms there, and the palettes would never be used, if I didn't manually move them, myself, but, it redownloads them, as well as the drivers and vhds and everything, every damn time I update, to the wrong directories.

This is not an issue that should be raised here. I suggest you open it up here instead https://github.com/MiSTer-devel/Downloader_MiSTer

And since you mention vhds and drivers, I recommend you name the title something other than "Palettes"

**Comment by sorgelig** [comment]
This issue has nothing to do with this repository.
Open issue in Downloader as thehughhefner said.

**Comment by Paradox** [comment]
That's not true, if palettes were moved to their own directories, MiSTer and the cores would need to be updated to read them from the new directory, and the updater would just start downloading/updating them in the new location. The rest of it might be something to bring up, in the updater repo, and I am already working on a pull request regarding that, but, it really has nothing to do with palettes unnecessarily being placed in the directories meant to hold the roms and other directly related files. Shadow masks and saves aren't in the roms directory...

**Comment by birdybro** [comment]
I apologize for making you feel insulted in any way. That was not my intent.

**Comment by Paradox** [comment]
> Palettes are core dependent and are supposed to be automatically downloaded into that core's folder, e.g.:
> 
> ```
> .../games/NES/Palettes/...
> ```
> 
> They are downloaded already as their own folder from here --> https://github.com/MiSTer-devel/Distribution_MiSTer/tree/main/games/NES/Palettes - and here --> https://github.com/MiSTer-devel/Distribution_MiSTer/tree/main/games/ATARI7800/Palettes - and here --> https://github.com/MiSTer-devel/Distribution_MiSTer/tree/main/games/GAMEBOY/Palettes
> 
> Additionally, the palettes' path begins inside the core's rom directory.
> 
> Since the vast majority of cores don't have loadable palettes, I don't see much benefit to changing this, and it would add lines of code and another variable to Main unnecessarily as a result.

So, tell my why this is different from cheats... why aren't cheats in the main rom directory for the cores, too?

**Comment by birdybro** [comment]
I dunno, ask the person in charge.
